README.md

```diff
@@ -20,6 +20,32 @@ sequenceDiagram
 ```
 
 
+# Objectives and Explanation of OAuth Status Assertions
+
+Status assertions are used to provide time sensitive claims that impact the validity of digital credentials without the need for a direct request from third parties. They are periodically issued to the holders of the credentials, who can present them to the verifiers together with the corresponding digital credentials.
+
+## Key aspects of the concept of OAuth Status Assertions:
+OAuth Status Assertions provide a robust method of verifying the validity of digital credentials without the need for verifiers to access external sources or third parties.
+
+### Purpose and goal: 
+
+The main goal of OAuth Status Assertions is to **confirm the non-revocation of a digital credential**. This allows verifiers to check the validity of a credential without having to access external systems or databases. This increases privacy, reduces latency and facilitates offline verification.
```

Comment:
The same comment about the used terminology as above i.e confirm the status of credential.

I would say that this "preserves" privacy instead of "increases". Increase is relative.

Reply:
Done